pkg/sql/sem/tree/testdata/pretty/create_table.align-deindent.golden

```diff
@@ -68,7 +68,7 @@ CREATE TABLE product_information (
 		INT8
 		REFERENCES customers_2 (
 			id
-		)
+		) MATCH FULL
```

Comment:
How were these not printed before? I don't see how the change to pretty.go would add this text.

Reply:
I added it to the input SQL :)